angular-app/src/app/app.component.html

```diff
@@ -0,0 +1,3 @@
+<app-shell></app-shell>
+<app-home></app-home>
+<app-todo-add></app-todo-add>
```

Comment:
Nu este gresit sa declaram tag-ul app-todo-add aici, in cazul notru. Dar pe viitor, cand o sa adaugam si alte pagini, atunci acest buton o sa apara in toate paginile.

O regula generala ce o putem urma, deoarece folosim module, ar fi sa declaram componentele fix unde le folosim. In cazul notru, app-todo-add ar trebui declarat in modulul home.

Reply:
Done.